### PR TITLE
added in support for access logging

### PR DESCRIPTION
**Pull request by snowman2**
## Description
Added in option for API Gateway access logging

## GitHub Issues
#1665

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -848,6 +848,9 @@ to change Zappa's behavior. Use these at your own risk!
         "cloudwatch_log_level": "OFF", // Enables/configures a level of logging for the given staging. Available options: "OFF", "INFO", "ERROR", default "OFF". C
         "cloudwatch_data_trace": false, // Logs all data about received events. Default false.
         "cloudwatch_metrics_enabled": false, // Additional metrics for the API Gateway. Default false.
+        "access_logging": false, // Add in logging of who accesses the API
```
Comment (jneves) [inline README.md]: Can you keep the configuration options in alphabetical order please?

Comment (snowman2) [reply README.md]: Sure, just made the change.

**Comment by snowman2** [comment]
This is used in our deployments & works well.

**Comment by snowman2** [comment]
Any updates on the possibility of merging this into master?

**Comment by snowman2** [comment]
Unrelated test failed (test_slim_handler): https://travis-ci.org/Miserlou/Zappa/jobs/643517448

All others passed.

**Comment by snowman2** [comment]
@jneves any blockers to merging this one?

**Comment by snowman2** [comment]
Switched to [serverless](github.com/serverless/serverless)

**Pull request closed without merging**